2200098/exam1.py

```diff
@@ -0,0 +1,15 @@
+# 최빈수 구하기 문제
+t = int(input())
+for _ in range(t) :
+    t_case = '#' + input()
+    cnt_dict = {}
+    num_list = list(map(int, input().split()))
+    for num in num_list :
+        cnt_dict[num] = (cnt_dict[num] + 1 if num in cnt_dict.keys() else 1)
+    max_cnt = max(cnt_dict.values())
+    max_list = []
+    for key, value in cnt_dict.items() :
+        if value == max_cnt : max_list.append(key)
+    print(t_case, sorted(max_list)[-1])
```

Comment:
깔끔한 풀이네요! 👍 
```suggestion
    print(t_case, max(max_list))
```
단순하게 최댓값 만을 구해도 된다면, `max` 함수를 이용하셔도 될 것 같습니다!